docs/derivations.md

```diff
@@ -4,6 +4,8 @@ You can use **purs-nix** to make a derivations for your PureScript bundle, your
 
 You access these derivations via the [modules](./purs-nix.md#purs-modules) attribute set, using the name of your module as the attribute.
 
+NOTE: A non-top-level module requires quotation marks around its module name (e.g. `modules."Foo.Bar.Main".bundle`)
```

Comment:
Can you change it to say something like "if the module name contains a `.`" instead of "non-top-level", since you can put `module Foo.Bar.Main` in `src/Main.purs` (as is commonly done with `Test.Main`).